### PR TITLE
Make the menu blendable

### PR DESCRIPTION
**Pull request by tomalec**
https://github.com/Starcounter/RebelsLounge/issues/368

Requires: https://github.com/Juicy/slot-all/pull/2, https://github.com/Starcounter/uniform.css/pull/163

### CODE REVIEW

File: src/UniformDocs/wwwroot/UniformDocs/MasterPage.html [hunk 1]
```diff
@@ -1,45 +1,59 @@
...
 <link rel="import" href="/UniformDocs/elements/github-source-links.html">
+<link rel="import" href="/sys/slot-all/slot-all.html">
 
 <template>
     <link rel="stylesheet" href="/UniformDocs/style.css">
     <dom-bind>
         <template id="root" is="dom-bind">
-            <imported-template href="{{model.NavPage.UniformDocs_0.Html}}"></imported-template>
+            <starcounter-include slot="uniformdocs/menu" view-model="{{model.NavPage}}">
+                <template is="declarative-shadow-dom" presentation="parent" preserve-content>
```
Comment (warpech) [inline src/UniformDocs/wwwroot/UniformDocs/MasterPage.html]: `preserve-content` - is this a performance optimization, or what's the reason?

Comment (tomalec) [reply src/UniformDocs/wwwroot/UniformDocs/MasterPage.html]: That's Polymer thing , not to erase `<template>` content when use in `<dom-bind>` (or Polymer element)

File: src/UniformDocs/wwwroot/UniformDocs/MasterPage.html [hunk 2]
```diff
@@ -1,45 +1,59 @@
...
+                            padding-top: 3px !important;
+                            padding-bottom: 3px !important;
+                        }
+                        nav.uniformdocs-nav > a {
+                            padding-top: 3px !important;
+                            padding-bottom: 3px !important;
+                        }
+                    </style>
+                    <nav class="uniformdocs-nav uni-vertical-nav">
+                        <slot-all></slot-all>
```
[the remark this answers is not in the document]
Comment (tomalec) [reply src/UniformDocs/wwwroot/UniformDocs/MasterPage.html]: >  the menu is not really blendable, because its handler is not registered in blend.json.

**Comment by tomalec** [comment]
> The only change request that I have is that the menu does not show up in Firefox:

That was fixed two days ago with https://github.com/Juicy/slot-all/pull/2

I was planning to release new SCF today, but there are still some issues with stability of uniform.css master branch

**Pull request merged**